modules/peers.js

```diff
@@ -225,7 +225,7 @@ __private.insertSeeds = function (cb) {
 		});
 	}, function (err) {
 		library.logger.trace('Peers->insertSeeds - Peers discovered', {updated: updated, total: library.config.peers.list.length});
-		return setImmediate(cb);
+		return setImmediate(cb, err);
```

Comment:
@4miners This code seems related to you. Was it intentionally that you missed sending error to top level chain?

Reply:
Few points:
- Fail to update status of single peer not means that function `insertSeeds` failed entirely
- Even if error is propagated here - it's discarded at top level function (`async.series`)

So adding error propagation here not change behavior.